packages/amplify-category-storage/src/index.ts

```diff
@@ -80,54 +107,54 @@ export async function getPermissionPolicies(context: $TSContext, resourceOpsMapp
         } else {
           permissionPolicies.push(policy);
         }
-        resourceAttributes.push({ resourceName, attributes, category: categoryName });
+        resourceAttributes.push({ resourceName, attributes, storageCategory });
       } else {
-        printer.error(`Provider not configured for ${categoryName}: ${resourceName}`);
+        printer.error(`Provider not configured for ${storageCategory}: ${resourceName}`);
       }
     } catch (e) {
-      printer.warn(`Could not get policies for ${categoryName}: ${resourceName}`);
+      printer.warn(`Could not get policies for ${storageCategory}: ${resourceName}`);
       throw e;
     }
-  }
+  });
 
   return { permissionPolicies, resourceAttributes };
 }
 
-export async function executeAmplifyCommand(context: $TSContext) {
+async function executeAmplifyCommand(context: any) {
   let commandPath = path.normalize(path.join(__dirname, 'commands'));
 
   if (context.input.command === 'help') {
-    commandPath = path.join(commandPath, categoryName);
+    commandPath = path.join(commandPath, AmplifyCategories.STORAGE);
   } else {
-    commandPath = path.join(commandPath, categoryName, context.input.command);
+    commandPath = path.join(commandPath, AmplifyCategories.STORAGE, context.input.command);
   }
 
-  const commandModule = await import(commandPath);
+  const commandModule = require(commandPath);
 
   await commandModule.run(context);
 }
 
-export async function handleAmplifyEvent(context: $TSContext, args: $TSAny) {
-  printer.info(`${categoryName} handleAmplifyEvent to be implemented`);
+async function handleAmplifyEvent(context: any, args: any) {
+  printer.info(`${AmplifyCategories.STORAGE} handleAmplifyEvent to be implemented`);
   printer.info(`Received event args ${args}`);
 }
 
-export async function initEnv(context: $TSContext) {
-  const { resourcesToBeSynced, allResources } = await context.amplify.getResourceStatus(categoryName);
+async function initEnv(context: any) {
+  const { resourcesToBeSynced, allResources } = await context.amplify.getResourceStatus(AmplifyCategories.STORAGE);
   const isPulling = context.input.command === 'pull' || (context.input.command === 'env' && context.input.subCommands[0] === 'pull');
-  let toBeSynced: $TSObject[] = [];
+  let toBeSynced = [];
 
   if (resourcesToBeSynced && resourcesToBeSynced.length > 0) {
-    toBeSynced = resourcesToBeSynced.filter((b: $TSObject) => b.category === categoryName);
+    toBeSynced = resourcesToBeSynced.filter((b: any) => b.category === AmplifyCategories.STORAGE);
```

Comment:
please use  a descriptive variable name instead of b

Reply:
This is not my change and is legacy code.